internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -991,7 +991,7 @@ func (c *ClusterManager) buildObservabilitySubscriptionResource() *v1alpha1.Subs
 			CatalogSource:          observabilityCatalogSourceName,
 			Channel:                "alpha",
 			CatalogSourceNamespace: observabilityNamespace,
-			StartingCSV:            "observability-operator.v3.0.9",
+			StartingCSV:            "observability-operator.v3.0.10",
```

Comment:
maybe its time we make this configurable via cli flags

Reply:
I can make a Jira to make this change. Thanks @machi1990